pkg/printers/internalversion/describe.go

```diff
@@ -2291,7 +2291,10 @@ func describeServiceAccount(serviceAccount *api.ServiceAccount, tokens []api.Sec
 			mountHeader: mountSecretNames,
 			tokenHeader: tokenSecretNames,
 		}
-		for _, header := range sets.StringKeySet(types).List() {
+
+		keys := sets.StringKeySet(types).List()
```

Comment:
List() already sorts

Reply:
Thanks @liggitt - saw it in http://velodrome.k8s.io/dashboard/db/bigquery-metrics?orgId=1 - looks like it takes a while to roll off and was not sure if this was already fixed.